### DISCUSSION TITLE
UI broken on iOS Safari

### DISCUSSION THREAD

**Discussion by scorsi** [community/community#66] [Discussions]
Hello there,
The UI is actually broken on iOS Safari as shown in the following screen.
![920DB59F-1261-4EF0-A4C1-C3600579A0F6](https://user-images.githubusercontent.com/8389441/99185648-83ab3c80-274b-11eb-9fe3-e6f57bcca3be.png)

**Comment by becca** [comment]
Thank you for this report, we'll have a fix out for this as soon as possible.
